addons/mail/__manifest__.py

```diff
@@ -2,7 +2,7 @@
 
 {
     'name': 'Discuss',
-    'version': '1.1',
+    'version': '1.3',
```

Comment:
Change is so breaking that we move it up 2 versions?
Or is it in preparation of another version change in another PR?